lib/tasks/populate_canonical_models.rake

```diff
@@ -88,12 +88,209 @@ namespace :canonical_models do
       end
     end
 
+    desc 'Populate or update canonical building and smithing materials from JSON data'
```

Comment:
These Rake tasks are kind of ugly but they're also [about to get significantly modified](https://trello.com/c/YdoBROVq/161-modify-rake-tasks-for-populating-canonical-items-to-clean-up-items-not-in-json-files), so I've decided to leave them as-is. I think they're reasonably readable so I'm deferring the refactor until I do the linked card.